.vsts-ci.yml

```diff
@@ -128,7 +134,7 @@ stages:
             artifactName: BuiltPackageOutputs
             artifactType: container
           condition: eq(variables['_BuildConfig'], 'Release')
-      
+        
```

Comment:
is this change needed?

Reply:
no, not sure how it happened. I removed the spaces